Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseHelper.swift

```diff
@@ -4,22 +4,12 @@ import PocketCastsUtils
 
 class DatabaseHelper {
     class func setup(queue: PCDBQueue) {
-        var startingSchemaVersion: Int32 = 0
-
-        queue.inDatabase { db in
+        queue.write { db in
```

Comment:
This method was split in two in one of the GRDB PRs as [explained here](https://github.com/Automattic/pocket-casts-ios/pull/2820/files#r1985476408).

This method is back to being a single block, which is what it should be. I had to add a method so that the crash that was happening before doesn't happen anymore.

With this change we're back at how it was before + working correctly with GRDB.